src/main/java/org/sonarlint/intellij/trigger/FileEditorTrigger.java

```diff
@@ -19,15 +19,20 @@
  */
 package org.sonarlint.intellij.trigger;
 
+import com.intellij.openapi.application.ApplicationManager;
 import com.intellij.openapi.components.AbstractProjectComponent;
+import com.intellij.openapi.fileEditor.FileEditor;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.intellij.openapi.fileEditor.FileEditor'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)